net/bulk_emulation/ut/bulk_mem_ut.c

```diff
@@ -675,10 +675,10 @@ static void test_failure(void)
 }
 
 enum {
-	PING_CLIENT_SEGMENTS = 8,
-	PING_CLIENT_SEGMENT_SIZE = 512,
-	PING_SERVER_SEGMENTS = 4,
-	PING_SERVER_SEGMENT_SIZE = 1024,
+	PING_CLIENT_SEGMENTS = 1,
+	PING_CLIENT_SEGMENT_SIZE = 4096,
+	PING_SERVER_SEGMENTS = 1,
+	PING_SERVER_SEGMENT_SIZE = 4096,
```

Comment:
Why is this needed?

Reply:
In case of libfabric, for RPC max seg_nr = 1 and max segment size = 1MB